src/jdk.crypto.ec/share/classes/sun/security/ec/SunEC.java

```diff
@@ -247,6 +255,10 @@ void putEntries() {
         putService(new ProviderServiceA(this, "AlgorithmParameters",
             "EC", "sun.security.util.ECParameters", apAttrs));
 
+        if (!FIPSConfigurator.enableFIPS()) {
```

Comment:
Again, the method name is wrong and the condition is inverted.

Reply:
modified